storage/innobase/xtrabackup/test/inc/common.sh

```diff
@@ -1309,6 +1309,34 @@ function innodb_checkpoint_lsn()
         grep "Last checkpoint at" | awk '{ print $4 }'
 }
 
+#general method to restore
+########################################################################
+function backup_and_restore() {
```

Comment:
why backup_restore is added again? I remember that you or marce added this already

Reply:
for some reason, It was not there. may be some mistake with final merge. Also there was issue in jenkin. It was not reporting the correct failures which Illia has fixed. So it didn't show up in jenkins.

Reply:
well, there is no GCA involved here.. are you working on old commit? Please update to latest 8.0 and retry.

"for some reason" has to be found :wink:

Reply:
the fix is on latest commi of 8.0 t. As I mention it was not caught in jenkin because of some issue which Illia has fixed.
anyway, the Jenkin is fixed, now we will come to know if any test is failing

Reply:
Sorry, I didn't understand completely. You will remove this duplicate code?

Reply:
backup_and_restore code is not in 8.0

Reply:
ok, got it. Then we will introduce as part of this PR